spec/dummy/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,3 @@
+ReactOnRails.configure do |config|
+  config.bundle_js_file = "app/assets/javascripts/generated/server.js"
```

Comment:
What if there will be more than 1 js app? I think it should be defined on `react_component` level.